OrbitTest/main_short_lived_threads.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <chrono>
+#include <iostream>
+#include <mutex>
+#include <random>
+#include <set>
+#include <string>
+#include <thread>
+#include <vector>
+
+std::mutex g_joinable_mutex;
+std::set<std::thread::id> g_joinable_threads;
+
+// Busy wait for ttl_ms.
+void Worker(int ttl_ms) {
+  auto deadline = std::chrono::system_clock::now() + std::chrono::milliseconds(ttl_ms);
+  while (true) {
+    if (std::chrono::system_clock::now() > deadline) {
+      break;
+    }
+  }
+  std::lock_guard lock(g_joinable_mutex);
+  g_joinable_threads.emplace(std::this_thread::get_id());
+}
+
+// Main thread goes into a busy loop that spawns and joins threads.
+// The spawned threads go into a busy wait.
+// The first command line parameter gives the number of threads to spawn.
+// The second command line parameter gives the live time of each spawned thread.
+// The actual live time is varied between 100% and 200% of that value to
+// make things slightly more interesting.
+int main(int argc, char* argv[]) {
+  if (argc != 3) {
+    std::cerr << "Usage:\n./OrbitTestShortLivedThreads number_of_threads time_to_live_per_thread\n";
```

Comment:
Maybe specify the time unit: time_to_live_per_thread_ms?

Reply:
yep, done.